Source/Events.Store.MongoDB/Processing/Streams/SubscriptionState.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.ApplicationModel;
+using Dolittle.Tenancy;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Dolittle.Runtime.Events.Store.MongoDB.Processing.Streams
+{
+    /// <summary>
+    /// Represents the state of an <see cref="SubscriptionState" />.
+    /// </summary>
+    public class SubscriptionState : AbstractSubscriptionState
```

Comment:
As I commented above, we could just put the stuff in AbstractSubscriptionState in here